app/src/main/java/net/gsantner/markor/ui/hleditor/TextActions.java

```diff
@@ -618,6 +620,20 @@ protected boolean runCommonTextAction(final @StringRes int action) {
                 runRenumberOrderedListIfRequired();
                 return true;
             }
+            case R.string.tmaid_common_insert_snippet: {
```

Comment:
suggestion: Long press button shows dialog, copying the selected portion of text (or all if no selection) into new snippet. aka  "create new snippet"

Reply:
I definitely want to keep the long - press for the last snippet. For example, I have `<input type="checkbox">`  which I would like to rapidly insert per-line.

Instead, we can do this via the neutral button in the dialog.